ports/atmel-samd/boards/pygamer/mpconfigboard.mk

```diff
@@ -0,0 +1,27 @@
+LD_FILE = boards/samd51x19-bootloader-external-flash.ld
+USB_VID = 0x239A
+USB_PID = 0x802E
+USB_PRODUCT = "PyGamer"
+USB_MANUFACTURER = "Adafruit Industries LLC"
+
+QSPI_FLASH_FILESYSTEM = 1
+EXTERNAL_FLASH_DEVICE_COUNT = 1
+EXTERNAL_FLASH_DEVICES = GD25Q64C
+LONGINT_IMPL = MPZ
+
+# No I2S on SAMD51G
+CIRCUITPY_AUDIOBUSIO = 0
```

Comment:
Should we turn this off? It's a SAMD51J. (Comment is misleading.)

Reply:
oops thats right ill fix - can you do the pybadge?

Reply:
ok i think done!